src/tactic/interactive.lean

```diff
@@ -1053,32 +1053,55 @@ example (i j k : ℕ) (h₀ : i ≤ j) (h₁ : j ≤ k) : i ≤ k :=
 begin
   extract_goal,
      -- prints:
-     -- example {i j k : ℕ} (h₀ : i ≤ j) (h₁ : j ≤ k) : i ≤ k :=
+     -- example (i j k : ℕ) (h₀ : i ≤ j) (h₁ : j ≤ k) : i ≤ k :=
      -- begin
-
+     --   admit,
      -- end
   extract_goal my_lemma
-     -- lemma my_lemma {i j k : ℕ} (h₀ : i ≤ j) (h₁ : j ≤ k) : i ≤ k :=
+     -- prints:
+     -- lemma my_lemma (i j k : ℕ) (h₀ : i ≤ j) (h₁ : j ≤ k) : i ≤ k :=
```

Comment:
Is the `{}` to `()` change here intended? Since it's not changed in the examples below, I'm guessing not.

Reply:
Oh, never mind, I just tried it out and it uses the same binders as the original statement. That sounds reasonable.